front/src/Bands.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { Container, Button, Card, Col, CardDeck } from "react-bootstrap";
+import userphoto from "./userphoto.png";
+
+export default class Bands extends Component {
+  constructor(props) {
+    super(props);
+    this.handlePage = this.handlePage.bind(this);
+    this.state = {
+      next: "http://127.0.0.1:8000/bands/",
+      previous: null,
+      bands: []
+    };
+  }
+
+  componentDidMount() {
+    const url = this.state.next;
+    this.handleUrl(url);
+  }
+
+  handleUrl(url) {
+    fetch(url)
+      .then(result => result.json())
+      .then(result => {
+        this.setState({
+          next: result.next,
+          bands: result.results,
+          previous: result.previous
+        });
+        console.log(result);
```

Comment:
Do we need pagination on newest bands? They are only 10 displayed always, so there is no need for pagination I think here.